__tests__/app.test.js

```diff
@@ -103,4 +103,57 @@ describe("/api/articles", () => {
 				expect(body.articles).toBeSortedBy("created_at", {descending: true});
 			});
 	});
-})
\ No newline at end of file
+})
+
+
+describe("/api/articles/:article_id/comments", () => {
+	it("GET: 200 - responds with a correct comments array", () => {
+		return request(app)
+			.get("/api/articles/3/comments")
+			.expect(200)
+			.then(({ body }) => {
+				body.comments.forEach((comment) => {
```

Comment:
Technically this test could provide a false positive as the forEach will not execute if an empty array is returned. 
Please also add an expect statement that checks the length of the comments array to prevent a false positive